core/pom.xml

```diff
@@ -471,9 +471,9 @@ THE SOFTWARE.
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
-    <dependency><!-- [JENKINS-4433] -->
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>log4j-over-slf4j</artifactId>
```

Comment:
I'm pretty sure there is something still transitively depending on log4j and that needs to be exclueded.